tests_new/integration_tests/modules/notifications/test_notifications.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from assertpy import assert_that
+
+from integration_tests.errors import GqlError
+from integration_tests.modules.notifications.queries import (
+    list_notifications,
+    count_deleted_notificiations,
+    count_read_notificiations,
+    count_unread_notificiations,
+    mark_notification_read,
+    delete_notification,
+)
+
+
+def test_list_notification(client1):
+    assert_that(list_notifications(client1)).is_not_none()
+    assert_that(list_notifications(client1, filter={'read': True})).is_not_none()
+    assert_that(list_notifications(client1, filter={'unread': True})).is_not_none()
+    assert_that(list_notifications(client1, filter={'archived': True})).is_not_none()
```

Comment:
Maybe we can assert the keys of the response, not only that it is not None

Reply:
asserted `contains_key(...)`